README.md

```diff
@@ -9,6 +9,7 @@ of a PHP library.
 
  * Your project uses `git`
  * Your project uses `composer.json` to define its dependencies
+ * Composer is aware of the class loading semantics of your project, see e.g. the `"autoload"` section of `composer.json`
```

Comment:
Not really composer, but just `composer.json`. let's rephrase this to state that all source paths are covered by an `autoload` section in `composer.json`

Reply:
here we go